.github/workflows/auto-pr.yaml

```diff
@@ -0,0 +1,97 @@
+name: Auto-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
+env:
+  TERM: dumb
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.CR_PAT }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # This is necessary to avoid unexpected auto-merge in git cherry-pick
+          fetch-depth: 0
+
+      - name: Create pull requests
+        run: |
```

Comment:
I feel it might be more maintainable if we create a script with the following code.
In that case, the script accepts all the `github.xxx` variables as arguments so that we can run the script in local. (maybe)
What do you think?

Reply:
Sounds good! I'll try it

Reply:
Addressed it in https://github.com/scalar-labs/scalardb/pull/884/commits/ce4e2b583d4319b3ed3ed4b81953b57fa5ce839d